lib/pwwka/configuration.rb

```diff
@@ -13,6 +13,7 @@ class Configuration
     attr_accessor :async_job_klass
     attr_accessor :send_message_resque_backoff_strategy
     attr_accessor :default_prefetch
+    attr_accessor :receive_raw_payload
```

Comment:
This new configuration option should be documented in the README.

Reply:
Yeah, I volunteered to do that, so if the overall feature looks good, I can write up the docs and make sure it passes the build matrix on cricle (since travis doesn't work)